--- analitza-17.04.3/analitza/expression.h.omv~	2017-07-13 00:42:17.526882601 +0200
+++ analitza-17.04.3/analitza/expression.h	2017-07-13 00:42:22.822755864 +0200
@@ -116,7 +116,7 @@ class ANALITZA_EXPORT Expression
         /**
          *    Returns the expression of the lambda body (without resolving the dependencies)
          */
-        Expression lambdaBody() const Q_REQUIRED_RESULT;
+        Expression lambdaBody() const;
         
         /**
          *    Returns whether it is a vector expression.
