diff -urN analitza-4.14.1/CMakeLists.txt analitza-4.14.1-patched/CMakeLists.txt
--- analitza-4.14.1/CMakeLists.txt	2014-07-13 04:14:14.000000000 +1100
+++ analitza-4.14.1-patched/CMakeLists.txt	2014-10-01 09:02:23.715679391 +1100
@@ -17,8 +17,11 @@
 macro_log_feature(CURSES_FOUND "NCurses" "Allows KAlgebra to provide a console interface." "http://www.gnu.org/software/ncurses/" FALSE "" "")
 
 macro_optional_find_package(OpenGL)
-macro_bool_to_01(OPENGL_FOUND HAVE_OPENGL)
-macro_log_feature(OPENGL_FOUND "OpenGL" "Support for 3D graphs in Analitza" "http://opengl.org" FALSE "" "")
+option(SHOULD_BUILD_OPENGL "Build OpenGL support" ON)
+if(SHOULD_BUILD_OPENGL)
+  macro_bool_to_01(OPENGL_FOUND HAVE_OPENGL)
+endif(SHOULD_BUILD_OPENGL)
+macro_log_feature(HAVE_OPENGL "OpenGL" "Support for 3D graphs in Analitza" "http://opengl.org" FALSE "" "")
 
 include_directories(.)
 add_definitions (-DQT_USE_FAST_CONCATENATION -DQT_USE_FAST_OPERATOR_PLUS)
